src/index/testing.rs

```diff
@@ -146,7 +159,7 @@ impl Context {
 
     for (id, entry) in runes {
       pretty_assert_eq!(
-        outstanding.get(id).copied().unwrap_or_default(),
+        outstanding.get(id).copied().unwrap_or_default() - burned_balances,
```

Comment:
Add assert above which asserts that burned_balances is correct, similar to the above asserts.